### DISCUSSION TITLE
Weekly Totals for New Cases, Deaths, Recoveries

### DISCUSSION THREAD

**Discussion by aoschwartz7** [reichlab/covid19-forecast-hub#2663] [Q&A] [answered]
Hi, is there a standard format for how model data is calculating weekly totals for new cases, deaths, and recoveries? Specifically, do weekly data totals get calculated starting on a Sunday, or Monday, etc?

**Comment by ydhuang28** [comment, marked as answer]
Hi! Please refer to [this section](https://github.com/reichlab/covid19-forecast-hub/wiki/Truth-Data#weekly-truth-data) of the Truth Data Overview page wiki for how we aggregate daily incident/total cases/deaths into weekly ones. We do not currently track recoveries.

**Reply by aoschwartz7** [reply]
Thanks!
